rasa/core/policies/rule_policy.py

```diff
@@ -489,3 +492,45 @@ def _default_predictions(self, domain: Domain) -> List[float]:
                 domain.index_for_action(self._fallback_action_name)
             ] = self._core_fallback_threshold
         return result
+
+    def persist(self, path: Union[Text, Path]) -> None:
+        """Persists `RulePolicy` to storage.
+
+        Args:
+            path: Path to persist policy to.
+        """
+        self.featurizer.persist(path)
+
+        memorized_file = Path(path) / "memorized_turns.json"
```

Comment:
does that actually overwrite what's saved by the MemoizationPolicy? Probably best to give it another name.

Reply:
we wouldn't because it's in its own directory (`policy_0_MemoizationPolicy` vs `policy_2_RulePolicy`)

Reply:
I'll rename it anyway, it's clearer